app/javascript/dashboard/components/layout/AvailabilityStatus.vue

```diff
@@ -20,22 +20,28 @@
               :key="status.value"
               class="status-items"
             >
-              <button
-                class="button clear status-change--dropdown-button"
-                :disabled="status.disabled"
+              <woot-button
+                variant="clear"
+                class-names="status-change--dropdown-button"
+                :is-disabled="status.disabled"
                 @click="changeAvailabilityStatus(status.value)"
               >
                 <availability-status-badge :status="status.value" />
                 {{ status.label }}
-              </button>
+              </woot-button>
             </woot-dropdown-item>
           </woot-dropdown-menu>
         </div>
       </transition>
 
-      <button class="status-change--change-button" @click="openStatusMenu">
+      <woot-button
+        variant="clear"
+        color-scheme="secondary"
+        class-names="status-change--change-button link"
```

Comment:
Can check if we need these classnames?

Reply:
Yes, "status-change--change-button" is used in spec.js  file.